src/main/webapp/util/ajax.js

```diff
@@ -199,16 +199,25 @@ export const DocumentHandler = {
 
 export const User = {
 
-  isCurrentUserAdmin(url) {
-    return axios.get(url);
+  getUserSession(url) {
+    return axios.get(url)
   },
 
-  addExtraProperties(url, projectKey, data) {
-    return axios.post(url+ '?id=' + projectKey, data );
+  getAllUsers(serverUrl, query) {
```

Comment:
If this utility could get access to the global `serverUrl`, that would cut down on a ton of boilerplate that we have now.

Reply:
I wasn't aware that ajax file had direct access to global constants. But after talking about it with the team, i realize that it is. I would leave this a minor tech debt since changing these (even if its simple) it would require a re-test of the entire application to save any possible bug.

Reply:
I wasn't sure either ... thanks for looking into that!